pkg/dataplane/item.go

```diff
@@ -85,20 +85,26 @@ func (i Item) GetFieldUint64(name string) (uint64, error) {
 }
 
 // For internal use only - DO NOT USE!
-func (i Item) GetShard() (map[int]*ItemChunk, *ItemCurrentChunkMetadata, error) {
+func (i Item) GetShard() (map[uint64]*ItemChunk, *ItemCurrentChunkMetadata, error) {
```

Comment:
It's better not to change  function return types
Why just convert uint64 to int? Map key can really get that big?

Reply:
you right, will convert it to int